commands/sim.js

```diff
@@ -41,16 +42,18 @@ module.exports = function (program, conf) {
     .option('--enable_stats', 'enable printing order stats')
     .option('--backtester_generation <generation>','creates a json file in simulations with the generation number', Number, -1)
     .option('--verbose', 'print status lines on every period')
+    .option('--silent', 'only output on completion (can speed up sim)')
```

Comment:
I'm thinking `--silent` and `--verbose` are slightly duplicate, in the sense that the current output with all of the lines showing exactly what and when the strategy does things is more useful for strategy developers (and thus should be considered verbose) where a consumer would only be interested in sim results.
What do you think about dropping `--silent` and moving showing output lines only if `--verbose` is set?

Reply:
So they currently do different things. The `--verbose` option turns on the status lines for sim and train that have a carriage return at the end, generating the tape of new entries on every period. The `--silent` option actually turns off the `stdout` output where you'll see the system overwriting the same line over and over again with the current status line (the bit thats always flashing during live/sim). I didn't want to have it default to one or the other as the nice new darwin output actually reads that stdout output in order to provide progress for individual sims, so the new hotness doesn't look quite as nice (it still gives number of active sims, just not an estimate on time).